README.md

```diff
@@ -1,5 +1,88 @@
 # How to run
-
-Run ```docker run -p 5432:5432 -e POSTGRES_PASSWORD=password -e POSTGRES_USER=user postgres``` before starting 
+Run ```docker run -p 5432:5432 -e POSTGRES_PASSWORD=password -e POSTGRES_USER=user postgres``` before starting
 the main Spring Boot application.
+# What is the application?
+REST API monolith with a postgres database. The application.yml
+    contains the app's configuration details, like Liquibase config for database
+    migration.
+
+### Endpoints
+
+1. **Get all customers**
+    - **Request:**
+        - Method: GET
+        - URL: `/api/v1/customers`
+        - Example Request:
+          ```
+          GET /api/v1/customers
+          ```
+
+    - **Response:**
+        - Status Code: 200 OK
+        - Body: Array of customer objects containing `id`, `name`, `email`, and `age` fields.
+
+2. **Add a new customer**
+    - **Request:**
+        - Method: POST
+        - URL: `/api/v1/customers`
+        - Example Request:
+          ```
+          POST /api/v1/customers
+          Content-Type: application/json
+   
+          {
+            "name": "John Doe",
+            "email": "john@example.com",
+            "age": 30
+          }
+          ```
+
+    - **Response:**
+        - Status Code: 201 Created
+        - Body: New Customer object containing `id`, `name`, `email`, and `age` fields.
+
+3. **Delete a customer**
+    - **Request:**
+        - Method: DELETE
+        - URL: `/api/v1/customers/{customerId}`
+        - Example Request:
+          ```
+          DELETE /api/v1/customers/123
+          ```
+
+    - **Response:**
+        - Status Code: 200 OK
+        - Body: Success message confirming deletion.
+
+4. **Update a customer**
+    - **Request:**
+        - Method: PUT
+        - URL: `/api/v1/customers/{customerId}`
+        - Example Request:
+          ```
+          PUT /api/v1/customers/123
+          Content-Type: application/json
+   
+          {
+            "name": "Updated Name",
+            "email": "updated@example.com",
+            "age": 35
+          }
+          ```
+
+    - **Response:**
+        - Status Code: 201 Created
+        - Body: New Customer object containing `id`, `name`, `email`, and `age` fields.
+
```

Comment:
Add a section here called "How the application works". Mention Liquibase (e.g. where the `migrations.xml` is), sequence diagram, Swagger configuration, the `application.yml`, SQL for the database etc...